app/views/providers/has_dependants/show.html.erb

```diff
@@ -1,6 +1,67 @@
-<%= page_template page_title: t('.page_heading'), template: :basic do %>
-  <%= render 'shared/forms/has_dependants_form',
-             dependant_info: t('.info'),
-             dependant_extra_info: t('.extra_info'),
-             form_path: providers_legal_aid_application_has_dependants_path(@legal_aid_application) %>
+<%= page_template page_title: t('.page_title'), template: :basic do %>
+  <% if @legal_aid_application.has_dependants? %>
+    <%= govuk_fieldset_header(size: 'm') { t('.existing', count: "#{pluralize(@legal_aid_application.dependants.count, 'dependant')}")} %>
+    <div class="govuk-summary-list">
+      <% @legal_aid_application.dependants.each do |dependant| %>
+        <div class="govuk-summary-list__row">
+          <dd class="govuk-summary-list__value"><%= dependant.name %></dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="<%= providers_legal_aid_application_dependant_path(@legal_aid_application, dependant) %>">Change<span class="govuk-visually-hidden"> name</span></a></dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="remove_2">
```

Comment:
Just curious what 'remove_2' in the href does?

Reply:
Good catch, I thought I'd deleted this entire block!😢  
There should never be any dependants when viewing this page so it was moved to the has_other_dependants show :+1: